### PR TITLE
Ks 2021 11 add token form to proposal list view

### PR DESCRIPTION
**Pull request by Rineee**
This adds a form for the voting token to the proposal list. When the token is there, it simply displayed for now, we will then need a react component for showing the left votes (I will next add that to the api). 
Oh, and we also need a widget for the token, so that it can be entered with 3 x 4 characters.

### CODE REVIEW

File: meinberlin/apps/budgeting/templatetags/react_proposals.py [hunk 1]
```diff
@@ -4,14 +4,21 @@
...
 
 @register.simple_tag(takes_context=True)
-def react_proposals(context, obj):
-    proposals_api_url = reverse('proposals-list', kwargs={'module_pk': obj.pk})
+def react_proposals(context, module):
+    proposals_api_url = reverse('proposals-list',
+                                kwargs={'module_pk': module.pk})
     attributes = {'proposals_api_url': proposals_api_url,
-                  'is_voting_phase': 'voting' in obj.active_phase.type}
+                  'is_voting_phase': has_feature_active(module,
```
Comment (Rineee) [inline meinberlin/apps/budgeting/templatetags/react_proposals.py:18]: @fuzzylogic2000 I fixed this for now, so we dont get the 500, but I think this should go to the api as has_voting_permission, so admins are always allowed to vote, right? What do we do with showing the ratings though? Should they be visible for admins all the time? So then also just show them with has_rating_permission?

Comment (khamui) [reply meinberlin/apps/budgeting/templatetags/react_proposals.py:18]: nice

File: meinberlin/apps/votes/models.py [hunk 2]
```diff
@@ -78,6 +80,11 @@ def is_valid(self, model):
     def is_valid_for_item(self, item):
         return item.module == self.module and self.is_valid(item.__class__)
 
+    def __str__(self):
+        return '{}-{}-{}'.format(self.token[0:4],
+                                 self.token[4:8],
+                                 self.token[8:13])
```
Comment (fuzzylogic2000) [inline meinberlin/apps/votes/models.py:86]: This should only go up to 12, `self.token[8:12]` or to the end `self.token[8:]`

**Review by fuzzylogic2000 — COMMENTED** [review]
Very nice! Just a tinsi tiny thing. :)

File: meinberlin/apps/votes/models.py (hunk 2 above)
Comment (Rineee) [reply meinberlin/apps/votes/models.py:86]: Haha, right, good catch!

**Review by fuzzylogic2000 — APPROVED** [review]

**Pull request merged**